### ISSUE TITLE
Mosaico not quite integrated with Civi 5.4.0

### ISSUE THREAD

**Issue by nc3man** [veda-consulting/uk.co.vedaconsulting.mosaico#277]
We tend to keep our core CiviCRM fairly well aligned with most current version (no comment, please, if you disagree with why we do that ;) ). I recently installed Mosaico despite the fact that compatibility is listed only for 4.6, 4.7 Civi versions and we are running Civi 5.4.0.  In our Civi backend, the Mosaico template builder shows up as a new item now, but we do not have something like "New Mailing (Mosaico)" so that we can actually use the templates that we build. Is the functionality there, located different than in say 4.7.31 (we participated in the CiviDesk webinar yesterday on Mosaico based on Civi 4.7.31 and saw there what we were missing).

**Comment by MegaphoneJon** [comment]
It sounds like you may be running Mosaico 1.x.  CiviDesk is almost certainly showing the Mosaico 2.x functionality.  Mosaico 2.0 is in beta, but many folks are using it in production, and it works great on CiviCRM 5.x.  Can you confirm your Mosaico version?  If you're missing those features and running 2.x we should follow up; otherwise I think you can upgrade and close this.

**Comment by nc3man** [comment]
I was able to install the latest development version 2.0-beta4.1528762072 on a test Civi instance of our website. On the surface, it looks correct, but am reluctant to plop this down on less-tech-savvy staff without really knowing this is OK. Btw, I installed via "cv dl --dev flexmailer shoreditch mosaico" per the README.md. Clean and simple. Is that a version you recommend? If there is a different beta version, what would be the "cv" command to get that (excuse me as I'm a relative newbie installing extensions manually via git)
